src/phoebus_driver.cpp

```diff
@@ -1068,6 +1069,10 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin) {
         Real gam1[3][3];
         Real gam2[3][3];
         Real gam3[3][3];
+        const Real z = coords.Xc<3>(k, j, i);
+        const Real sigma = 2 * coords.CellWidthFA(X3DIR, k, j, i);
+        const Real s0 = s * std::exp(-z * z / sigma / sigma);
```

Comment:
Alternatively (or possibly additionally) we should probably normalize our Gaussian

Reply:
done